.remarkrc

```diff
@@ -32,6 +32,10 @@
     [
       "remark-preset-lint-node/remark-lint-nodejs-links.js",
       false
+    ],
+    [
+      "remark-lint-unordered-list-marker-style",
```

Comment:
This is the style used in the org and API docs, that's why it's set in the `remark-preset-lint-node`

Reply:
Gotcha. Going to undo this. It was generating a few warnings for the recently touched files, and I was wondering why it was not complaining before 🤔

Reply:
Most of those rules should eventually go on, but there are a few that are incompatible. It's been awhile, and I don't recall if that JSON-ish config file actually supported comments, so it's unfortunatly not obvious

Reply:
Prettier using remark to do it's formatting, but unfortunately doesn't read from the config file, so you may be getting conflicts between the 2

Reply:
BTW, adding the prettier preset is also just overriding the org style

Reply:
> BTW, adding the prettier preset is also just overriding the org style

I checked the docs, and the prettier preset is actually used to remove the conflicts that prettier has by default. (Like Prettier doesn't understand what Frontmatter is... It should not conflict with our rules afaik)

Reply:
It is disabling rules/config that are explicitly turned on by the Node preset. They may conflict with prettier, but that is Node markdown styleguide (today). If you want to be more prettier compatible in the org, you can suggest changes in https://github.com/nodejs/remark-preset-lint-node, but those changes also need to be make to the API docs in nodejs/node

Reply:
Are you sure we have conflicting rules? Where do you see the conflicts? 👀

Reply:
The unorderd list style is the first one I saw, but you'd need to do a crosswalk of https://github.com/nodejs/remark-preset-lint-node/blob/main/index.js (including the remark-preset-lint-recommended list) to see the full conflict.
Easiest way might be to move the prettier plug-in load before node one too see the Node styles re-enabling all the rules it has disabled, but in the end, not using the plug-in is the answer

Reply:
Question: If I set the remark-lint-node rule below the prettier one, the remark-lint-node will override prettier rules, right?